### PR TITLE
Change base image to rancher/agent-base

### PR DESCRIPTION
**Pull request by galal-hussein**

### CODE REVIEW

**Comment by leodotcloud** [comment]
@galal-hussein can you please split the changes into two commits, the first one being a base copy and the second one with the actual changes. It's hard to figure out what's being changed.

File: etcd/containers/0.13.1/etcd/build [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+ACCT=${ACCT:-rancher}
```
Comment (leodotcloud) [inline etcd/containers/0.13.1/etcd/build:3]: Just to be in sync with other microservices, can you please use `REPO` instead of `ACCT`?

**Comment by leodotcloud** [comment]
Thinking more about this, what if we move the code from this repo to `https://github.com/rancher/rancher-etcd`? Create a v2 branch in that repo and put in the code there so that both v2 and v3 are in the same place?

**Comment by galal-hussein** [comment]
@leodotcloud Closed in favor of https://github.com/rancher/rancher-etcd/pull/3

**Pull request closed without merging**